docs/src/getting-started/first-steps.md

```diff
@@ -46,11 +46,19 @@ your protocol buffers as a package.
 Let us define a dependency of the webserver on a hypothetical library
 called `user` in the `datatypes` repository.
 
-This is done by editing the manifest or by invoking `buffrs add`:
+This is done by invoking `buffrs add`:
+
+```bash
+$ buffrs add --registry https://your.registry.com datatypes/user@=0.1.0
+```
+
+The result is a dependency in the `Proto.toml`:
 
```

Comment:
Can we add a `> Note: Using the exact operator `=` is currently required when specifying dependencies` – maybe phrased better, but to explain why `@=0.1.0` is necessary?